commons-component-upgrade/src/main/java/org/exoplatform/commons/upgrade/UpgradeProductPlugin.java

```diff
@@ -216,11 +216,6 @@ public boolean shouldProceedToUpgrade(String newVersion, String previousGroupVer
             || VersionComparator.isSame(newVersion, previousArtifactVersion)))) {
       return false;
     }
-    // If the plugin has a target version that is before current version

```

Comment:
Isn't it the expected behavior ? why don't we change the target to be after the previos version or/and update the VersionComparator to handle other cases (tribe specific versionning with dates ) ?

Reply:
modified, can you review @ahamdi ?